locust/main.py

```diff
@@ -350,10 +350,15 @@ def main():
         sys.exit(0)
 
     locustfile = find_locustfile(options.locustfile)
+
     if not locustfile:
         logger.error("Could not find any locustfile! Ensure file ends in '.py' and see --help for available options.")
         sys.exit(1)
 
+    if locustfile == "locust.py":
+        logger.error("The locustfile must not be named `locust.py`. Please rename the file and try again.")
```

Comment:
No I mean if someone is naming their file `locust.py`, seems like they don't have many locustfiles and are just looking for whatever "default" name works, in which case `locustfile.py` suits that need. But I guess it can be misinterpreted. Let's leave it then.